extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -129,7 +130,7 @@ public static function onOasisSkinAssetGroups( &$jsAssets ) {
 			$jsAssets[] = self::ASSET_GROUP_LIFTIUM_EXTRA;
 		}
 
-		if ( $wgAdDriverUseGoogleConsumerSurveys && WikiaPageType::getPageType() === 'article' ) {
+		if ( $wgAdDriverUseGoogleConsumerSurveys && $pageType === 'article' ) {
```

Comment:
Is assigning `$pageType === 'article'` result to `$isPageAnArticle` variable and then reusing it in two conditional over-engineering? ;)